### ISSUE TITLE
Label format customization

### ISSUE THREAD

**Issue by dmfst** [nline/nline-plotlyjs-panel#24]
Thank you for maintaining this awesome plugin. 
I switched some of my old visualizations created with 'ae3e-plotly-panel' and works perfectly. The only difference observed is that the label format on mouse over is always with black font on white background. 
Is there a way of changing back the background and font color of the label to preserve the marker color?

**Comment by jacksongoode** [comment]
Yes, you can see the reference docs for all the settings [here](https://plotly.com/javascript/reference/layout/#layout-hoverlabel).

We have the default set to a white background for the hover but you can change that and the font color to whatever you want by adding the config in the `layout` section.

```
hoverlabel:
  bgcolor: blue
  font:
    color: red
```

Let me know if that resolves your issue.

**Comment by dmfst** [comment]
Thank you for the hint. It works for all series. 
I found up that it works also for each marker by passing the same list of individual colors like this:
````
        marker: { 
			colors: marker_colors,
			},
        hoverlabel: { 
			bgcolor: marker_colors
			},
````
